auth_saml/__manifest__.py

```diff
@@ -4,13 +4,13 @@
 
 {
     "name": "SAML2 Authentication",
-    "version": "16.0.1.0.2",
+    "version": "17.0.1.0.2",
     "category": "Tools",
     "author": "XCG Consulting, Odoo Community Association (OCA)",
     "maintainers": ["vincent-hatakeyama"],
     "website": "https://github.com/OCA/server-auth",
     "license": "AGPL-3",
-    "depends": ["base_setup", "web"],
+    "depends": ["base_setup", "web", "auth_oauth"],
```

Comment:
Do not add the dependency to `auth_oauth`, the module should be independent but compatible.

Reply:
Ok, it was because of the ordering in the config view, which was already there in 16.0 but gave me an error now. Removed the dependency.